frontend/src/metabase/admin/datamodel/metadata/components/MetadataTableList/MetadataTableList.styled.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import { color } from "metabase/lib/colors";
+
+interface AdminListItemProps {
+  selected?: boolean;
+  disabled?: boolean;
+}
+
+export const AdminListItem = styled.a<AdminListItemProps>`
+  padding: 0.75em 1em 0.75em 1em;
+  border: var(--border-size) var(--border-style) transparent;
+  border-radius: var(--default-border-radius);
```

Comment:
Safe to say this can be hard-coded as well
```suggestion
  border-radius: 8px;
```

I couldn't find any usage of these CSS custom properties except `var(--default-font-family)` which we can assume we don't use CSS custom properties (`--custom-properties` thing) in styled-components anymore.